### ISSUE TITLE
Object "Call get" graph node missing output slot?

### ISSUE THREAD

**Issue by legbo** [Vahera/godot-orchestrator#111]
### Describe the bug

Object class function "Call get" graph nodes don't have any output data slot.

![Screenshot_20240228_170232](https://github.com/Vahera/godot-orchestrator/assets/16309645/20ce1c5f-063b-4971-bbc9-6ac03d0665a4)

Object class 
get function must return Variant (Any)

![Screenshot_20240228_172103_Get](https://github.com/Vahera/godot-orchestrator/assets/16309645/546a728d-0477-40ee-8f03-fe11ad77ddbb)

### Expected behavior

I wanted to get custom property from my Resource or Node, but Object class "Call get" node don't have any output data slots.

![Screenshot_20240228_171753_Ex](https://github.com/Vahera/godot-orchestrator/assets/16309645/8d61ce10-6a3f-41d1-bb61-80dc7f0c0b8a)

### Actual behavior

You can set target Resource or Node and define what property value you want to get, but there is no output data slots.

### How to Reproduce?

1. Add Object -> "Call get" node from search

### Godot full version

v4.2.1

### Orchestrator version

2.0.dev3

**Comment by Naros** [comment]
Thanks @legbo, confirmed.

**Comment by NullMember** [comment]
I've experienced the same thing without knowing this issue existed. In my case, I've tried to call (using callv) some function from GDScript. I've looked into it, and if a function's return value is unknown, its return type is set to Variant::NIL and the Orchestrator doesn't create an output pin. However, there should be a output pin marked as 'any' because it's hard to know if the function has any return value or what its type is. It's possible to mark known functions (especially Godot's internal functions) if they are returning anything or nothing, but it's almost impossible to mark functions from outside. I think even if functions return nothing, an output pin should be created. This is the unfortunate nature of dynamically typed languages.
